GTBitsOfGood/brain-exercise-analytics
# Tyler/7 implement account recovery and password reset screens

## Implement Account Recovery and Password Reset Screens

Issue Number(s): #7 

What does this PR change and why?

Create `account-recovery` page and `reset-password` page to set up for users to input their information.

### Checklist

- [x] `/auth/account-recovery` page is implemented according to the Figma design
- [x] `/auth/reset-password` page is implemented according to the Figma design
- [x] `/auth/reset-password` is a dynamic route. Also, grab the token.
- [x] Clicking the buttons correctly redirect to the required pages
- [x] Errors are displayed as per the Figma design

### Critical Changes

- new `assets` and `components` folder
- new `account-recovery` and `reset-password` pages with CSS files

### Related PRs

- N/A

### Testing

1. Go to http://localhost:3000/auth/account-recovery to check out the `account-recovery` page
2. Click the `continue` button without putting anything in the input boxes, there should be an error message in red at the top.
3. Then, fill the input boxes and click the `continue` button, it should bring you to the "Email has been sent!" screen

1. Go to http://localhost:3000/auth/reset-password/[token] (you can put whatever you want instead of [token] and you don't need to have brackets, for example http://localhost:3000/auth/reset-password/random would work)
2. Click the `Back to Sign In` button and the `Sign Up Now` buttons, they should redirect you to their respective pages
